x-pack/platform/plugins/shared/streams/server/routes/streams/crud/route.ts

```diff
@@ -123,17 +130,21 @@ export const editStreamRoute = createServerRoute({
       throw badRequest('A group stream name can not start with [logs.]');
     }
 
-    const body = isGroupStreamDefinitionBase(params.body.stream)
-      ? {
+    const body: StreamUpsertRequest = isGroupStreamDefinitionBase(params.body.stream)
```

Comment:
do we have a `GroupStreamDefinitionBase` and a schema? I'm not sure if I understand why. `base` is something that implies it can be extended but I don't think we have an extension of a group stream.